types/uuid-parse/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for uuid-parse 1.0
+// Project: https://github.com/zefferus/uuid-parse
+// Definitions by: Christian Rackerseder <https://github.com/screendriver>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+import 'node';
```

Comment:
I think you want `/// <reference types="node" />` here instead.

Reply:
Hi @rbuckton ,
I changed that now but could you explain me please the difference because in this repository I see `import ...` as well as `/// <reference ...` and both seems to work 🤔

Reply:
One actually imports a module called `"node"` at runtime (i.e. adds emit for an import which isn't what you want), whereas `/// <reference types="node" />` just tries to find `@types/node` in `node_modules` and adds those types to the compilation.